### ISSUE TITLE
minTopDistance not working

### ISSUE THREAD

**Issue by triki97** [AndreaMiotto/PartialSheet#78]
I set min top distance and it doesnt seem to take effect. Xcode 12.0 beta 4. I can set the other properties of the style however and they work

**Comment by styrken** [comment]
Hello. Does it work in the final Xcode release?

**Comment by triki97** [comment]
Ive been trying in Xcode-beta-12.4 but not working there.

**Comment by schwjustin** [comment]
I created `top` and `bottom` variables to use throughout my app to account for safe area insets.
https://gist.github.com/schwjustin/f51a0d921b7b8e064005d8e85cce288d

Then, in PartialSheetViewModifier.swift I changed `let topInset` in `private var sheetPosition: CGFloat` from `UIApplication.shared.windows.first?.safeAreaInsets.top ?? 20.0` to `top`.

Changing `minTopDistance` now works as expected (for my application).
